README.md

```diff
@@ -25,12 +25,23 @@ Check out our [roadmap](ROADMAP.md) as we push towards a __0.1.0__ release.
 ## Using Kedge
 
 ### Installation
-You can download latest master build for all major operating systems and architectures.
-- [Linux (amd64)][Bintray Latest Linux]
-- [macOS (darwin)][Bintray Latest macOS]
-- [Windows (amd64)][Bintray Latest Windows]
+You can download latest binary (built on each master PR merge) for [Linux (amd64)][Bintray Latest Linux], [macOS (darwin)][Bintray Latest macOS] or [Windows (amd64)][Bintray Latest Windows] from [Bintray](https://bintray.com):
 
-Or you can build it yourself from the master GitHub branch:
+```sh
+# Linux 
+curl -L https://dl.bintray.com/kedgeproject/kedge/latest/kedge-linux-amd64 -o kedge
```

Comment:
In https://github.com/kedgeproject/kedge/issues/162#issuecomment-315756807, I meant something like what minikube does here - https://github.com/kubernetes/minikube#linux

Basically an all in one command, which downloads, renames, sets permissions and puts in $PATH